tasks/add-fbc-contribution/add-fbc-contribution.yaml

```diff
@@ -76,24 +79,43 @@ spec:
         build_tags=$(jq '.fbc.buildTags // []' ${DATA_FILE})
         add_arches=$(jq '.fbc.addArches // []' ${DATA_FILE})
         hotfix=$(jq -r '.fbc.hotfix // "false"' ${DATA_FILE})
+        pre_ga=$(jq -r '.fbc.preGA // "false"' ${DATA_FILE})
         staged_index=$(jq -r '.fbc.stagedIndex // "false"' ${DATA_FILE})
+        product_name=$(jq -r '.fbc.productName // ""' ${DATA_FILE})
+        product_version=$(jq -r '.fbc.productVersion // ""' ${DATA_FILE})
         build_timeout_seconds=$(jq -r --arg build_timeout_seconds ${default_build_timeout_seconds} \
             '.fbc.buildTimeoutSeconds // $build_timeout_seconds' ${DATA_FILE})
-        target_index=$(jq -r '.fbc.targetIndex' ${DATA_FILE})
         fbc_fragment=$(jq -cr '.components[0].containerImage' ${SNAPSHOT_PATH})
 
         timestamp_format=$(jq -r '.fbc.timestampFormat // "%s"' ${DATA_FILE})
         timestamp=$(date "+${timestamp_format}")
 
+        # default index_suffix and targetIndex
+        index_suffix=""
+        target_index=$(params.targetIndex)
+
+        # the targetIndex should be modified based on the hotfix or pre-GA
+        # information the `data.fbc` in release_plan_admission offers
         if [ "${hotfix}" == "true" ]; then
           issue_id=$(jq -r '.fbc.issueId // empty' ${DATA_FILE})
           if [ -z "${issue_id}" ]; then
             echo "Hotfix releases requires the issue id."
```

Comment:
issue id set in the fbc.issueId key of the ReleasePlanAdmission spec.data field

Reply:
block comment rewrote.

Reply:
Sorry, I don't think I was clear. I was talking about amending this echo statement with what I wrote `Hotfix releases requires the issue id set in the fbc.issueId key of the ReleasePlanAdmission spec.data field` or something like what you did on line 107

Reply:
Oh, I see. Ok, fixed.